.changeset/tall-eggs-own.md

```diff
@@ -0,0 +1,5 @@
+---
+'@blocto/blocto-wagmi-connector': patch
```

Comment:
Don't need to patch it if you only update the README file

Reply:
thx for remind, its removed.